Index: Zend/zend_arg_defs.c
===================================================================
RCS file: /repository/ZendEngine2/zend_arg_defs.c,v
retrieving revision 1.7
diff -u -r1.7 zend_arg_defs.c
--- Zend/zend_arg_defs.c	31 Dec 2007 07:12:06 -0000	1.7
+++ Zend/zend_arg_defs.c	20 Apr 2008 17:38:29 -0000
@@ -18,6 +18,8 @@
 
 /* $Id: zend_arg_defs.c,v 1.7 2007/12/31 07:12:06 sebastian Exp $ */
 
+#include "Zend_API.h"
+
 ZEND_BEGIN_ARG_INFO(first_arg_force_ref, 0)
 	ZEND_ARG_PASS_INFO(1)
 ZEND_END_ARG_INFO();
